agentops/decorators.py

```diff
@@ -344,14 +338,7 @@ def new_init(self, *args, **kwargs):
                         session=session,
                     )
                 except AttributeError as e:
-                    Client().add_pre_init_warning(
-                        f"Failed to track an agent {name} because agentops.init() was not "
-                        + "called before initializing the agent with the @track_agent decorator."
-                    )
-                    logger.warning(
-                        "Failed to track an agent. This often happens if agentops.init() was not "
-                        "called before initializing an agent with the @track_agent decorator."
-                    )
+                    print(e)
```

Comment:
We should still use the logger and pre_init_warning system. This piece could be used inside of a library like crew and cause printing to happen without agentops.init()

Reply:
you're right. i changed the wording so it's still applicable